### ISSUE TITLE
Resource access errors

### ISSUE THREAD

**Issue by greatislander** [accessibility-exchange/platform#753]
I've noticed two errors related to static resources on staging.accessibilityexchange.ca and future.accessibilityexchange.ca:

1. The following resource returns a 401 error:
   - https://staging.accessibilityexchange.ca/manifest.webmanifest
   - https://future.accessibilityexchange.ca/manifest.webmanifest

They are served from the [`public`](https://github.com/accessibility-exchange/platform/tree/staging/public) directory within the app.

2. There are two font files (`.woff2`) which are meant to be loaded from `storage/fonts` but are failing to load because of an SSL error related to the URL on future.accessibilityexchange.ca. They appear to be being served from https://iris-accessibility.v2.kube.colab.coop.

**Comment by greatislander** [comment]
Resolved via #766.
